projects/js-packages/components/components/threats-data-views/index.tsx

```diff
@@ -116,18 +112,48 @@ export default function ThreatsDataViews( {
 			titleField: THREAT_FIELD_TITLE,
 			mediaField: THREAT_FIELD_ICON,
 			showMedia: true,
-		},
-	};
+		};
+
+		const tableLayout = {
+			...baseView,
+			fields: [ THREAT_FIELD_SEVERITY, THREAT_FIELD_TYPE, THREAT_FIELD_AUTO_FIX ],
+			titleField: THREAT_FIELD_TITLE,
+			descriptionField: THREAT_FIELD_DESCRIPTION,
+			showMedia: false,
+		};
+
+		return isSm ? { list: listLayout } : { table: tableLayout, list: listLayout };
```

Comment:
Is it necessary to prevent mobile users from accessing the table layout like this?